float64 ya
float64 yb
float64 yc
float64 za
float64 zb
float64 zc
---
bool success